src/osparc_python_runner/main.py

```diff
@@ -97,34 +62,29 @@ def ensure_requirements(code_dir: Path) -> Path:
 
     else:
         requirements = requirements[0]
+        logger.info(f"Found: {requirements}")
     return requirements
 
+def _ensure_output_subfolders():
+    for n in range(1, NUM_OUTPUTS+1):
+        output_sub_folder_env = f"OUTPUT_{n}"
+        output_sub_folder = OUTPUT_FOLDER / OUTPUT_SUBFOLDER_TEMPLATE.format(n)
+        output_sub_folder.mkdir(parents=True)
+        logger.info("ENV defined for subfolder: $%s=%s", output_sub_folder_env, output_sub_folder)
 
-def setup():
-    logger.info("Cleaning output ...")
-    clean_dir(output_dir)
-
-    # TODO: snapshot_before = list(input_dir.rglob("*"))
-
-    # NOTE The inputs defined in ${INPUT_FOLDER}/inputs.json are available as env variables by their key in capital letters
-    # For example: input_1 -> $INPUT_1
-    #
-
-    logger.info("Processing input ...")
-    unzip_dir(input_dir)
-
-    # logger.info("Copying input to output ...")
-    # copy(input_dir, code_dir)
-
-    logger.info("Searching main entrypoint ...")
-    main_py = ensure_main_entrypoint(input_dir)
-    logger.info("Found %s as main entrypoint", main_py)
 
-    logger.info("Searching requirements ...")
-    requirements_txt = ensure_requirements(input_dir)
+def setup():
+    _ensure_output_subfolders()
+    logger.info("Processing input from %s:", INPUT_FOLDER)
+    logger.info("%s", list(INPUT_FOLDER.glob("*")))
```

Comment:
I think it would be more informative to have simply a ``os.system``. See this example
```cmd
>> import os
>> os.system("ls -la")
total 20
drwxrwxr-x 3 crespo crespo 4096 Sep 15 18:28 .
drwxrwxr-x 3 crespo crespo 4096 Jun 24  2021 ..
-rw-rw-r-- 1 crespo crespo 1312 Sep 15 18:28 input_script_1.py
drwxrwxr-x 2 crespo crespo 4096 Sep 15 18:22 __pycache__
-rw-rw-r-- 1 crespo crespo   20 Sep 15 18:10 requirements.txt
0
```
you can build a context manager to capture the stdout or use something like ``subprocess.run`` instead of ``os.system``